rebasehelper/application.py

```diff
@@ -439,6 +440,9 @@ def build_packages(self):
             results_dir = os.path.join(self.results_dir, version)
             build_dict['builds_nowait'] = self.conf.builds_nowait
             build_dict['build_tasks'] = self.conf.build_tasks
+            build_dict['enable_option'] = self.conf.enable_option
+            if self.conf.buildtool in ("copr", "fedpkg"):
+                logger.warning("We are not supporting option --enable-option for builder copr or fedpkg.")
```

Comment:
Why this is even displayed? It will be displayed every time when user uses copr or fedpkg without even using the new option. This should be completely removed.